src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -400,64 +400,65 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId }) => {
-
+  console.log(bedPresets[0]?.meta);
   return (
     <div className="flex h-[calc(100vh-1.5rem)] flex-col px-2">
       <div className="flex flex-wrap items-center justify-between gap-2">
-        <div className="flex items-center gap-4 px-3">
+        <div className="flex w-full items-center justify-between gap-4 px-3">
           <p className="block text-lg font-medium"> Camera Presets :</p>
-          <div className="flex items-center">
-            {bedPresets?.map((preset: any, index: number) => (
-              <button
-                key={preset.id}
-                onClick={() => {
-                  setLoading(CAMERA_STATES.MOVING.GENERIC);
-                  // gotoBedPreset(preset);
-                  absoluteMove(preset.meta.position, {
-                    onSuccess: () => {
-                      setLoading(CAMERA_STATES.IDLE);
-                      setCurrentPreset(preset);
-                      console.log(
-                        "onSuccess: Set Preset to " + preset?.meta?.preset_name
-                      );
-                      triggerGoal("Camera Preset Clicked", {
-                        presetName: preset?.meta?.preset_name,
-                        consultationId,
-                        userId: authUser.id,
-                        result: "success",
-                      });
-                    },
-                    onError: () => {
-                      setLoading(CAMERA_STATES.IDLE);
-                      setCurrentPreset(preset);
-                      console.log(
-                        "onError: Set Preset to " + preset?.meta?.preset_name
-                      );
-                      triggerGoal("Camera Preset Clicked", {
-                        presetName: preset?.meta?.preset_name,
-                        consultationId,
-                        userId: authUser.id,
-                        result: "error",
-                      });
-                    },
-                  });
-                  getCameraStatus({});
-                }}
-                className={classNames(
-                  "block border border-gray-500 px-4 py-2 first:rounded-l last:rounded-r",
-                  currentPreset === preset
-                    ? "border-primary-500 bg-primary-500 text-white"
-                    : "bg-transparent"
-                )}
-              >
-                {preset.meta.preset_name || `Preset ${index + 1}`}
-              </button>
-            ))}
+          <div className="z-30 flex flex-wrap items-center justify-center bg-gray-100 ">
+            <SelectFormField
+              id="preset"
+              name="preset"
+              label=" Preset"
+              errorClassName="hidden"
+              options={bedPresets?.map((preset: any) => preset)}
+              optionLabel={(preset: any) => preset?.meta?.preset_name}
+              optionValue={(preset: any) => preset}
+              placeholder={currentPreset?.value?.meta?.preset_name as string}
```

Comment:
```suggestion
              value={currentPreset}
```

Reply:
@rithviknishad  umm i guess its working and i have used `SelectFormField` . kindly let me know if i have to make any changes